script/openqa-load-templates

```diff
@@ -86,10 +86,10 @@ try {
     $info = Cpanel::JSON::XS->new->relaxed->decode($datafile);
     dd $info;
 }
-catch {
-    $info = eval $datafile;
-    if (my $error = $@) { die "Error in data file: $error\n" }
-};
+catch ($e) {
+    try { $info = Cpanel::JSON::XS->new->relaxed->decode($datafile) }
+    catch ($e) { die "Error in data file: $e\n" }
```

Comment:
That should be left an `eval` for now. This is a "string eval", which evaluates the given string as perl code. It's quite different from the "block eval", which is more like `try`.

Reply:
plus, you are basically doing the same thing as in the try block here, so why should this succeed the second time, if it fails the first time.